### PR TITLE
Add isopen(::Server)

### PR DESCRIPTION
**Pull request by jamii**
This used to work but disappeared somewhere in the last few versions.

### CODE REVIEW

**Comment by jamii** [comment]
Looks like TLS failures on nightly?

File: src/HttpServer.jl [hunk 1]
```diff
@@ -405,6 +405,9 @@ end
 """ Closes the Server `server` by closing the underlying TCPServer `sock` in the HttpHandler. """
 close(server::Server) = close(server.http.sock)
 
+""" A Server is open if the underlying TCPServer `sock` is open. """
```
Comment (hustf) [inline src/HttpServer.jl]: 'server' is open if it accepts new connections.

**Comment by hustf** [comment]
To :408. Assuming 
```
    servertask = @schedule run( server, portno)
```
...client connections established...
```
    close(server)
```
- the established client connections remains open
- websocket connections (if present) remains open
- any new connections are refused
- se.http.sock.status = 6

This is not the only intuitive behaviour of close(server). This may be why isopen(server) was removed?

**Comment by jamii** [comment]
I don't have strong feelings about it either way. I use `isopen(server.http.sock)` to check if I have already closed a server (partly because closing an already-closed server throws an exception).

**Pull request closed without merging**